### DISCUSSION TITLE
JIT mode and re generate config file

### DISCUSSION THREAD

**Discussion by karimelsaidy** [jaredh159/tailwind-react-native-classnames#252] [Q&A]
Is there an intention or plan to add a JIT MODE and re generate the config file on run time or add a way to request that during running time?

**Comment by jaredh159** [comment]
I'm not sure I'm clear on exactly what you're asking.`twrnc` essentially is always in JIT mode, it doesn't pre-compile anything ahead of time, it dynamically parses tailwind utilities at runtime, and that's the only mode it has.

Can you expand on what you're trying to do or what problem you're facing?
